pkg/sql/logictest/testdata/logic_test/serializable_eager_restart

```diff
@@ -29,13 +32,17 @@ SELECT * FROM t
 user root
 
 # The insert increases the candidate timestamp, but a restart won't occur
-# until an EndTransaction.
+# until RELEASE SAVEPOINT.
```

Comment:
The `EndTransaction` comment was good - it talks about when the error is detected by crdb at the kv layer. It is detected when an `EndTransaction` request is sent. The `EndTransaction` will indeed be sent by a `RELEASE SAVEPOINT`, but that doesn't need to be explained necessarily.
I'd revert this line.